DataFormats/GEMRecHit/interface/ME0Segment.h

```diff
@@ -72,7 +73,10 @@ class ME0Segment final : public RecSegment {
 
   float time() const    { return theTimeValue; }
   float timeErr() const { return theTimeUncrt; }
-    
+
+  float deltaPhi() const { return theDeltaPhi; }
+  static float computeDeltaPhi(const ME0Chamber * chamber, const LocalPoint& position, const LocalVector& direction );
```

Comment:
@nickmccoll - which class represents RecoMuonProducer? Duplicating the code is not a good idea, nor introducing dependency on a plugin.